### ISSUE TITLE
Unused Loss Functions: self.vqa_criterion and self.qav_criterion

### ISSUE THREAD

**Issue by gitimposs** [caicch/ColPro#1]
Hi, thanks for releasing the code — it's been very helpful for understanding your work.

While going through the implementation, I noticed that the loss functions _self.vqa_criterion_ and _self.qav_criterion_ are defined but do not seem to be used anywhere in the current training or evaluation pipelines. At the same time, _the adapter_ is not used. I’m wondering if the code is currently in a debugging?

Any clarification would be appreciated!

Thanks again for the great work!

**Comment by caicch** [comment]
Hi, thank you for your message, and I’m glad to hear the code has been helpful.
Yes, the code is currently in a debugging phase after switching to a new machine. I’ve uploaded the complete version, but you may need to adjust some hyperparameters to reproduce the exact results for now.
